core/math/math.odin

```diff
@@ -48,6 +51,57 @@ clamp :: builtin.clamp
 @(require_results) sqrt_f32be :: proc "contextless" (x: f32be) -> f32be { return #force_inline f32be(sqrt_f32(f32(x))) }
 @(require_results) sqrt_f64le :: proc "contextless" (x: f64le) -> f64le { return #force_inline f64le(sqrt_f64(f64(x))) }
 @(require_results) sqrt_f64be :: proc "contextless" (x: f64be) -> f64be { return #force_inline f64be(sqrt_f64(f64(x))) }
+/*
+Return square root of given input.
+
+**Only accept floats**
+
+Inputs:
+- x: input value of type floats
```

Comment:
Although `pkg.odin-lang` might do this conversion automatically, you should respect the LSP and wrap inputs and return value names in backticks, like this:

```markdown
- `x`: Input value.
```

Also please form a proper sentence when describing inputs and returns of a procedure. Meaning, capitalizing the first letter and putting a period at the end.

You don't have to specify that `x` in this function is a float, because you already specified this in the Note above. The point of this section is to establish a connection between the function parameters and its description.